pkg/updater/resultstore/resultstore.go

```diff
@@ -79,31 +77,6 @@ func extractGroupID(tg *configpb.TestGroup, result *processedResult) string {
 	// P - build info
 	case tg.GetPrimaryGrouping() == configpb.TestGroup_PRIMARY_GROUPING_BUILD:
 		return identifyBuild(tg, result)
-	// F - date (YYYY-mm-dd)
-	case tg.GetFallbackGrouping() == configpb.TestGroup_FALLBACK_GROUPING_DATE:
-		timing := result.InvocationProto.GetTiming().GetStartTime()
-		return time.Unix(timing.Seconds, int64(timing.Nanos)).UTC().Format("gid-2006-01-02")
-	// F - labels (hash)
-	case tg.GetFallbackGrouping() == configpb.TestGroup_FALLBACK_GROUPING_LABELS:
-		h := fnv.New32a()
-		for _, l := range result.InvocationProto.GetInvocationAttributes().GetLabels() {
-			h.Write([]byte(l))
-		}
-		return "gid-" + strconv.Itoa(int(h.Sum32()))
-	// F - ID
-	case tg.GetFallbackGrouping() == configpb.TestGroup_FALLBACK_GROUPING_ID:
-		return result.InvocationProto.GetId().GetInvocationId()
-	// F - build info (again)
-	case tg.GetFallbackGrouping() == configpb.TestGroup_FALLBACK_GROUPING_BUILD:
-		return identifyBuild(tg, result)
-	// F - configuration value
-	case tg.GetFallbackGrouping() == configpb.TestGroup_FALLBACK_GROUPING_CONFIGURATION_VALUE:
-		for _, property := range result.InvocationProto.GetProperties() {
-			if property.GetKey() == tg.GetFallbackGroupingConfigurationValue() {
-				return property.GetValue()
-			}
-		}
-		return ""
 	default:
 		return ""
```

Comment:
for default behaviour here we need to return invocationID
`result.InvocationProto.GetId().GetInvocationId()`

Reply:
done.